modules/ROOT/pages/stats-prometheus.adoc

```diff
@@ -0,0 +1,100 @@
+= Integrate Prometheus
+:description: Integrating Sync Gateway and Prometheus for Stats Monitoring and Alerts
+
+include::partial$_std-hdr-sgw.adoc[]
+
+:topic-group: Statistics
+:param-related:  {xref-sgw-pg-rest-api-metrics}  |  {xref-sgw-pg-stats-monitoring}
+:param-abstract: This content explains how to integrate Sync Gateway and Prometheus to provide effective monitoring and alerts for Sync Gateway events
+include::partial$block-abstract.adoc[]
+
+== Introduction
+
+Sync Gateway's Metrics REST API exposes stats in a {glos-term-prometheus} compatible format.
+The `metricsInterface` is off by default; to enable it see: <<lbl-prom-act>>.
+
+[#lbl-prom-act]
+== Activation
+
+include::partial$prometheus-activation.adoc[]
+
+== Integration
+
+You will need to integrate Sync Gateway's metrics feed with your Prometheus deployment.
+Couchbase provide both a configuration file and a sample rules file to make this integration with Prometheus easier.
+
+Copy both the Prometheus configuration file (`prometheus.yml`) file and the baseline rules directory (`rules/sync-gateway.rules.yml`) from Sync Gateway's release package into Prometheus's `/etc` directory.
+
+.Files in-situ
+====
+[source, bash]
+----
+/etc/prometheus/prometheus.yml // <.>
+/etc/prometheus/rules/sync-gateway.rules.yml // <.>
+----
+<.> You can change this location by specifying the path using the command line flag `--config.file` when starting Prometheus
+<.> You can specify a different location for the rules file by editing
+the path in the `rule_files` section of the `prometheus.yml` configuration file.
+====
+
+See also our blog entry {url-blog-prometheus}.
+
+== Configuration
+
+Configuration of Prometheus to work with Sync Gateway is governed by two files, starter copies of which are provided with Sync Gateway.
+
+Prometheus Configuration File::
++
+--
+The provided `prometheus.yml` file specifies the configuration required to scrape the Sync{nbsp}Gateway metrics target.
+It defines Sync Gateway's `metricsInterface` as being accessible on sync_gateway:4986/_metrics.
+If you have multiple Sync Gateways, you can either statically specify all their endpoints or configure Prometheus to discover the targets.
```

Comment:
Not too sure exactly what this bit means. If you want prometheus to find multiple targets you have to specify them as targets in the yml file.

Reply:
Removed discovery reference